pkg/kubelet/container/runtime.go

```diff
@@ -94,6 +94,8 @@ type Runtime interface {
 	RemoveImage(image ImageSpec) error
 	// Returns Image statistics.
 	ImageStats() (*ImageStats, error)
+	// Returns the filesystem path of the pod's network namespace
+	GetNetNS(containerID ContainerID) (string, error)
```

Comment:
Is it essential for adding `GetNetNS()` to `Runtime` interface? Not all container runtimes (e.g., HyperContainer, which is a hypervisor-based container runtime) running containers inside network namespace.